Sources/PackageDescription/PackageDescription.swift

```diff
@@ -27,50 +27,60 @@ import Foundation
 /// dependencies, and other configuration options.
 ///
 /// By convention, you need to define the properties of a package in a single
-/// nested initializer statement. Don’t modify it after initialization. The
+/// nested initializer statement. Don't modify it after initialization. The
```

Comment:
As mentioned previously, U+2019 is the correct character for punctuation, but U+0027 is more convenient. I don't hold a strong opinion to either one, other than that in general, the choice should be consistent in the entire package.

Reply:
I had thought I changed all of those, but maybe I missed some, or I somehow reintroduced them? I'll be more vigilant.

Reply:
Oh, never mind, I misunderstood your comment. Yes, consistency is desired, and I think straight back-ticks are the way to go for this current use case.